public/mixed_mich_2017/meta_study.txt

```diff
@@ -0,0 +1,7 @@
+type_of_cancer: mixed
+cancer_study_identifier: mixed_mich_2017
+name: Mixed Metastasis 500 (Michigan U 2017)
```

Comment:
Should we rename the study from `Mixed Metastasis 500 (Michigan U 2017)` to `Metastatic Solid Cancers (UMich, Nature 2017)`

Reply:
@rmadupuri updated.